vimiv/version.py

```diff
@@ -42,8 +42,8 @@ def info() -> str:
         f"Qt: {QT_VERSION_STR}\n"
         f"PyQt: {PYQT_VERSION_STR}\n\n"
         f"Svg Support: {bool(QtSvg)}\n"
-        f"Pyexiv2: {exif.pyexiv2.__version__ if exif.pyexiv2 is not None else None}\n"
-        f"Piexif: {exif.piexif.VERSION if exif.piexif is not None else None}"
+        f"Pyexiv2: {metadata.pyexiv2.__version__ if metadata.pyexiv2 is not None else None}\n"
```

Comment:
This line causes the linter to fail (too long). Whats is the best was to fit it? To use a variable ro the piexiv version?

Reply:
I would just break it like this:
```python
        f"Svg Support: {bool(QtSvg)}\n"
        "Pyexiv2: "
        f"{metadata.pyexiv2.__version__ if metadata.pyexiv2 is not None else None}\n"
        f"Piexif: {metadata.piexif.VERSION if metadata.piexif is not None else None}"
```